easybuild/easyconfigs/c/CalculiX/CalculiXCrunchiX-2.7-goolf-1.5.14.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Authors::   Bart Verleye
+# License::   
+#
+##
+easyblock='MakeCp'
+
+name = "CalculiXCrunchiX"
```

Comment:
the EasyBuild robot mechanism is going to have trouble finding the easyconfig in that case, so I would just throw it in a dedicated `CalculiXGraphiX` instead...